sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -5,6 +5,12 @@
 #include "azure/core/azure.hpp"
 #include "azure/core/http/http.hpp"
 
+#ifdef POSIX
+#include <poll.h> // for poll()
+#endif
+#ifdef WINDOWS
```

Comment:
_WIN32?

Reply:
we have a CMacke module to inject platfotm:
```cmacke
if (WIN32 OR MINGW OR MSYS OR CYGWIN)
  add_compile_definitions(WINDOWS)
endif ()
if (UNIX)
  add_compile_definitions(POSIX)
endif ()
```